app/views/home/index.html.erb

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+<head>
```

Comment:
why we need to add head tag here? what is the purpose of layout file,
https://github.com/zhamza372/EmployeeManagementSystem/pull/6/files#diff-9599427925097c3c66f26ac1e0de5cadR1